scout/server/blueprints/omics_variants/templates/omics_variants/outliers.html

```diff
@@ -109,7 +109,12 @@
                 {% endif %}
               </td>
               <td class="text-end">{{ '%.3e' % variant.p_value }}</td>
-              <td>{{ case.individuals.0.display_name }}</td>
+              <td>{% for sample in variant.samples %}
+                  {% if sample.genotype_call != "./." %}
```

Comment:
Nice with this check, could we add it also in the variants export code? https://github.com/Clinical-Genomics/scout/blob/34944ee153c33e14a57931f044f6f4bdf6ec74e8/scout/server/blueprints/omics_variants/controllers.py#L105

Reply:
Ah, right forgot - will check tomorrow!

Reply:
A GT check would be a placeholder as it stands now, with no GTs delivered from the pipeline. It will likely not be forgotten if or when we get such GTs.